compiler/next/lib/parsing/chpl.ypp

```diff
@@ -670,16 +670,13 @@ stmt:
   deprecated_decl_stmt      { $$ = context->finishStmt($1); }
 | include_module_stmt       { $$ = context->finishStmt($1); }
 | block_stmt                { $$ = context->finishStmt($1); }
+| executable_stmt
```

Comment:
Could you add a comment here that we're not calling `->finishStmt($1)` to avoid clearing stored comments?

Reply:
Yep, did this and reordered so that this doesn't break up the symmetry of the actions.